source/collaborate/collaborate-within-connected-microsoft-teams.rst

```diff
@@ -68,5 +68,5 @@ Chat notifications are sent in real-time whenever you're not active in Microsoft
 - Web browser: Mattermost delivers notifications when you've not had activity in your Microsoft Teams browser tab for 5 minutes or more, or when the browser tab is closed.  
 - Desktop app: Mattermost delivers notifications when you've not had activity at your computer for 5 minutes or more, or when the Microsoft Teams desktop app is closed.
 
-In order to avoid double notifications, Mattermost won't deliver chat notifications when your :ref:`Mattermost availability <preferences/set-your-status-availability:set your availability>` is set to **Available**, **Busy**, **Do not disturb**, or **Be right back** in Microsoft Teams.
```

Comment:
@esethna - Curious to know why we'd want to remove a link to a relevant docs page.

Reply:
Mattermost availability does not determine if the MS Teams notifications will be delivered, what matters is your availability in Microsoft Teams:

![image](https://github.com/user-attachments/assets/ae48812f-d9ef-468f-bb24-2f34b09b5464)

Reply:
Ah, of course! Thanks so much, @esethna!